build-system/pr-check/build-targets.js

```diff
@@ -231,7 +194,7 @@ function determineBuildTargets(buildTargets, fileName = 'build-targets.js') {
         .join(', ')
     )
   );
-  return areValidBuildTargets(buildTargets, fileName);
+  return true;
```

Comment:
This function no longer needs to return a boolean since we aren't doing a check. Can you change this function to no longer take a `Set<string>` param, but instead, simply create and return one?

After that, you can replace all instances of this code in `build-system/`

https://github.com/ampproject/amphtml/blob/bb89fd810594e5e9db4586dd3c4d7aebc915161b/build-system/pr-check/build.js#L55-L60

and this code

https://github.com/ampproject/amphtml/blob/68ce698f4563f766c4fac34bf86d23609d8007bd/build-system/pr-check/e2e-tests.js#L49-L50

with
```js
 const buildTargets = determineBuildTargets(FILENAME);
```

Reply:
done. removed a lot of code :-)